### PR TITLE
Bug 2042536: Fix SCC control

### PR DESCRIPTION
**Pull request by ArangoGutierrez**
Signed-off-by: Carlos Eduardo Arango Gutierrez <carangog@redhat.com>

### CODE REVIEW

File: config/manager/kustomization.yaml [hunk 1]
```diff
@@ -12,5 +12,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: quay.io/openshift/origin-cluster-nfd-operator
-  newTag: 4.11.0
+  newName: quay.io/eduardoarango/cluster-nfd-operator
```
Comment (wabouhamad) [inline config/manager/kustomization.yaml:15]: @ArangoGutierrez your private quay repo is listed here, should it be changed to the openshift quay repo ?

Comment (ArangoGutierrez) [reply config/manager/kustomization.yaml:15]: ups ;P

**Review by wabouhamad — COMMENTED** [review]
/lgtm

**Pull request merged**